js/job.js

```diff
@@ -1,26 +1,19 @@
+(function(context){
+  context.jobs = [];
 
-var jobs = [];
+  function Job (opt) {
+    this.company = opt.company;
+    this.position = opt.position;
+    this.dates = opt.dates;
+    this.responsibilities = opt.responsibilities;
+  }
 
-function Job (opt) {
-  this.company = opt.company;
-  this.position = opt.position;
-  this.dates = opt.dates;
-  this.responsibilities = opt.responsibilities;
-}
-
-var retrieveJobHistory = function(){
-  $.getJSON('https://raw.githubusercontent.com/tetsaiga86/Portfolio/class-07/data/jobHistory.json', function(data){
-    localStorage.jobHistory = JSON.stringify(data);
-    data.forEach(function(ele) {
-      jobs.push(new Job(ele));
+  context.retrieveJobHistory = function(){
+    $.getJSON('https://raw.githubusercontent.com/tetsaiga86/Portfolio/class-07/data/jobHistory.json', function(data){
+      localStorage.jobHistory = JSON.stringify(data);
+      data.forEach(function(ele) {
```

Comment:
So you have an array of generic objects that you want to convert into an array of Job objects...do we have any useful array methods here? (yes)